spec/2019-11-15-schnorrmultisig.md

```diff
@@ -44,14 +44,11 @@ Mode 2 (new Schnorr support, M-of-N; consumes N+M+3 items from stack):
 
 ## Triggering and execution mechanism
 
-Note that the above description leaves some ambiguity -- how does one know whether to execute in mode 1 or mode 2?
-Execution of the new Schnorr mode 2 is triggered by first doing a quick scan of the content of public keys and signatures. The new mode executes when:
+Whether to execute in mode 1 or mode 2 is detemined by the value of the checkbits element.
+* If the checkbits element is NULL, then Mode 1 is executed
+* If the checkbits element is non-NULL, then Mode 2 is executed.
 
-- The upgrade is active based on MTP; and
-- Every public key is encoded properly under current encoding rules: 33 byte compressed key starting with 0x02 or 0x03, or 65 byte uncompressed key starting with 0x04; and
-- Every signature is null (0-length) or correctly encoded Schnorr (65 bytes long, with a valid hashtype byte under current encoding rules).
-
-If any of the above conditions is not satisfied, then execution proceeds in the legacy mode where behaviour is unchanged from before.
+In practice "0-of-N" behavior is indistinguishable between the two modes.
```

Comment:
0-of-N would always have to be spend in legacy mode right now, since the dummy element would need to be zero.

Reply:
OK, yeah, but other than weird pubkey-validity quirks, the behavior of both modes should be the same for this case.

I will reword it just to avoid any possible ambiguity.